CHANGELOG-developer.next.asciidoc

```diff
@@ -92,6 +92,7 @@ The list below covers the major changes between 7.0.0-rc2 and main only.
 - Make winlogbeat/sys/wineventlog follow the unsafe.Pointer rules. {pull}36650[36650]
 - Cleaned up documentation errors & fixed a minor bug in Filebeat Azure blob storage input. {pull}36714[36714]
 - Fix copy arguments for strict aligned architectures. {pull}36976[36976]
+- Fix filebit gcs input panic {pull}38407[38407]
```

Comment:
This entry should be added to `CHANGELOG.next.asciidoc`. The "developer" file is focused mainly on API changes or internal-only fixes (without a user impact).

`filebit -> filebeat`

Reply:
I will create a PR to shift this around, since auto merge was enabled and it got merged.

Reply:
https://github.com/elastic/beats/pull/38512